float64[] x
float64[] y
float64[] yaw
float64[] s_d
bool direct_control
float64 direct_target_speed
bool emergency_stop
